creator-node/scripts/start.sh

```diff
@@ -64,9 +67,18 @@ if [ -z "$dbUrl" ]; then
 fi
 
 if [[ "$devMode" == "true" ]]; then
-    ./node_modules/.bin/nodemon --inspect=0.0.0.0:${debuggerPort} src/index.js | tee >(logger) | ./node_modules/.bin/bunyan
+    if [ "$link_libs" = true ]
+    then
+        cd ../audius-libs
```

Comment:
should this bit be in service-commands `init-repos.sh`?

Reply:
see comment above. i dont think we'd want to move it into init-repos.sh